frontend/src/modules/application/actions.js

```diff
@@ -24,4 +28,17 @@ export const createNewApplication = payload => async dispatch => {
             )
         }
     }
-}
\ No newline at end of file
+}
+
+// action used by applicant positions view
+export const fetchPositionsSuccess = payload => ({ type: FETCH_POSITIONS_SUCCESS, payload })
+export const fetchPositions = () => async dispatch => {
+    const res = await fetch("/api/v1/positions")
+    const data = await res.json()
```

Comment:
This needs to be in a `try .. catch` block. `await` turns failed promises into thrown errors.